picard/ui/searchdialog/artist.py

```diff
@@ -94,6 +95,10 @@ def parse_artists(self, artists):
             artist = Metadata()
             artist_to_metadata(node, artist)
             artist['score'] = node['score']
+            if 'disambiguation' in node:
+                artist['~artistcomment'] = node['disambiguation']
+            else:
+                artist['~artistcomment'] = ''
```

Comment:
Could this be simplified?

```python
artist['~artistcomment'] = node.get('disambiguation', '')
```

Reply:
Yes, much better, thanks for the suggestion.
Done in https://github.com/metabrainz/picard/pull/2664/commits/f3e3d3e1587d9c1d8aae6db325a147f5efeca920